rest_framework/fields.py

```diff
@@ -954,6 +956,10 @@ class FloatField(Field):
     def __init__(self, **kwargs):
         self.max_value = kwargs.pop('max_value', None)
         self.min_value = kwargs.pop('min_value', None)
+        self.localize = kwargs.pop('localize', False)
+        self.coerce_to_string = kwargs.pop('coerce_to_string', None)
```

Comment:
Why not `False` here?

Reply:
Same reason as in the previous point.

Reply:
Can you just make this `False` (and in the docs signature too) — I think it's cleaner. Thanks!